pkg/kv/kvserver/testing_knobs.go

```diff
@@ -95,6 +96,9 @@ type StoreTestingKnobs struct {
 	// or data.
 	TestingRangefeedFilter kvserverbase.ReplicaRangefeedFilter
 
+	// When an err is returned in a batch response call this hook
+	TestingServerResponseError func(context.Context, *roachpb.BatchRequest, error)
```

Comment:
Can you move this next to here:

https://github.com/cockroachdb/cockroach/blob/7051448458ce86c4ed8d9224921ae4ffd65bdac5/pkg/kv/kvserver/testing_knobs.go#L80-L83

and also name it symmetrically, i.e. something like `TestingResponseErrFilter`?

Reply:
Sounds good thanks!